### DISCUSSION TITLE
Please add an "example" section in the website...

### DISCUSSION THREAD

**Discussion by sylvaticus** [marp-team/marp#545] [Ideas]
Before looking at _how_ to do something with a tool, I am generally interested in _what_ I can do with it.
As a first-time user, I would have appreciated a set of pair examples (source/compiled PDF) from the basic to the most esoteric ones, in order to get a clue on what it is possible to do with this framework....

thank you for developing it !

**Comment by yhatt** [comment]
Currently `marp.app` has an example section like following.

[<img src="https://github.com/user-attachments/assets/483da9d1-6c98-4f7b-8a4d-39989bda9da3" alt="Example section of marp.app" width="500" />](https://marp.app)

For now, as mentioned in https://github.com/orgs/marp-team/discussions/503#discussioncomment-11494053, the following slide example by Marp community may be helpful.

- **Marp Basic Example** by @yhatt 
  - [Gist](https://gist.github.com/yhatt/a7d33a306a87ff634df7bb96aab058b5)
  - [Speaker Deck](https://speakerdeck.com/yhatt/marp-basic-example)
